### PR TITLE
[156] Container names set to cfp_v3_{container}_1

### PR DESCRIPTION
**Pull request by ngiersz**

### CODE REVIEW

**Comment by w1stler** [comment]
Why are we changing that? ;-)

**Comment by ngiersz** [comment]
Container names was sometimes created randomly (e.g. codeforpoznanpl_v3_backend_1 or code_for_poznan_pl_v3_backend_1 or codeforpoznanplv3_backend_1) and they were not coherent with these in Makefile

File: docker-compose.yml [hunk 1]
```diff
@@ -3,6 +3,7 @@ version: "3"
 services:
 
   frontend:
+    container_name: cfp_v3_frontend_1
```
Comment (w1stler) [inline docker-compose.yml]: We can remove `_1` from here as well as in other places.

File: Makefile [hunk 2]
```diff
@@ -13,13 +13,13 @@ stop:	## Stop the environment
 	docker-compose stop
 
 bash:	## Go to the backend container
-	docker exec -ti codeforpoznanpl_v3_backend_1 bash
+	docker exec -ti cfp_v3_backend_1 bash
```
Comment (w1stler) [inline Makefile]: Here we can use `docker-compose exec` with the namees `frontend`, `backend`, etc. Just please test it. ;-)

**Comment by w1stler** [comment]
Fixes #156

**Pull request merged**